### PR TITLE
sprint 1 issue 5 resolved

### PR DESCRIPTION
**Pull request by camilledaugherty**
This pull request resolves issue #5.

What was changed: A landing page for our classroom interface was created. This includes a header, footer, and vertical navigation menu to link to future pages.
Why it was changed: We needed a specific page for our interface, as using the default TPEN page would not fit our specifications. These changes will provide a base webpage for us to develop our site and make things simple for users.
How it was changed: I added a classroom folder inside components with two files: an html file with our landing page's code, and a css file with our page's styles. Layout pictured below:
image

### CODE REVIEW

**Review by sarah-fidahussain-SLU — CHANGES_REQUESTED** [review]
The header and the footer look correct when I ran the index.html file. However, I believe the vertical pane should be left-justified and not be below the header, rather to the left of the header and the footer. I included an image of what the vertical pane seems to look like based on a web search.

image

**Review by sarah-fidahussain-SLU — CHANGES_REQUESTED** [review]
Once I added a link to the html that links the css file, this is the result I am getting: 
Screenshot 2024-09-16 at 5 24 58 PM

Looks good to me now, but I think the gray box is not overlaying the home tab fully.

**Comment by kate-holdener** [comment]
> This pull request resolves issue #5.
> 
> What was changed: A landing page for our classroom interface was created. This includes a header, footer, and vertical navigation menu to link to future pages. Why it was changed: We needed a specific page for our interface, as using the default TPEN page would not fit our specifications. These changes will provide a base webpage for us to develop our site and make things simple for users. How it was changed: I added a classroom folder inside components with two files: an html file with our landing page's code, and a css file with our page's styles. Layout pictured below: image

Please use:
Fixes #5 
This will automatically close the issue, when this pull request is merged.

File: _site/default-transcribe/index.html
Comment (kate-holdener) [inline _site/default-transcribe/index.html:1]: @camilledaugherty I see that you deleted three files from the _site/default_transcribe directory. Looking at the main branch of your repo, I don't see a _site/default_transcribe directory. From which branch did you create your branch?

**Review by MutyaSowmya123 — CHANGES_REQUESTED** [review]
->Link the styles.css in the HTML page
->Check the styles that apply correctly to the elements
   
I noticed that the files in the _site folder were deleted. Just a reminder that while these files are typically generated and not tracked in version control, it’s important to ensure that the original source files are preserved.

File: _site/default-transcribe/index.html
Comment (camilledaugherty) [reply _site/default-transcribe/index.html:1]: @kate-holdener I just created it from the main branch. I'm not quite sure what those files are and I don't remember making any changes, please let me know if that is a problem!

Comment (MutyaSowmya123) [reply _site/default-transcribe/index.html:1]: @camilledaugherty The `_site` directory typically contains generated files from a static site generator. It often includes HTML, CSS, and JavaScript files that are built from your source content. I think deleting these files doesn't create a problem for now. But in future ensure the original source files are preserved.

**Comment by camilledaugherty** [comment]
requested changes have been made :)

**Review by sarah-fidahussain-SLU — APPROVED** [review]

**Review by MutyaSowmya123 — APPROVED** [review]

**Pull request merged**